### ISSUE TITLE
Documentation of Utility Nav is wrong

### ISSUE THREAD

**Issue by AlexVanK** [OER-WEB-TEAM/drupal-subtheme--nihod#43]
Currently the utility nav documentation assumes a new website to setup search with default search module -- in which case a template is provided and we're all good.

But in the case of custom search (e.g.: elastic search) a custom search bar needs to be created as a block and placed in the proper region.

Documentation needs to cover both scenarios.

---

Also the template name for the utility nav is wrong. There is no such thing as "Utiliy Nav" on Drupal, it's called Tools nav.
Rename template accordingly and describe setup in documentation

**Comment by AlexVanK** [comment]
Also consider moving utility nav into right place when installing theme.
These should be possible with Drupal setup config system (yml)

**Comment by emsaiz** [comment]
Ok I have some slight iterations to the current utility nav. We might need more if we still don't like these but I didn't want to do anything too extra if we don't need it. 

Adobe XD Link: https://xd.adobe.com/view/124158b0-1037-4ee2-b081-aef1353d537a-4712/

NOTE: I tried **restructuring the page info hierarchy** to account for the two different types of set ups. I made the "Responsive" and "Customization" sections their own H2s but maybe those instructions just pertained to the default search module? Anyways, feel free to disregard! 

NOTE 2: "Customization" section is repeated from "Paginator" component.  

![image](https://github.com/OER-WEB-TEAM/design-system--drupal-theme/assets/59488566/6ed3515d-9324-4b6f-bd05-aa7a0d0cd3bf)
